### ISSUE TITLE
Create "smart" OBS_GetBoundariesByBBox methods

### ISSUE THREAD

**Issue by andrewxhill** [CartoDB/observatory-extension#38]
Basically, take a ZOOM number and give back a "correct" polygon set based on both Coverage and Scale.

So at a global zoom, it would give back countries
At a regional zoom, counties
City zoom, blocks

cc @ohasselblad @talos

**Comment by ohasselblad** [comment]
So for a center point and a zoom level, generate a bounding box for that view and return all geoms in that bounding box?

**Comment by andrewxhill** [comment]
No, depending on the bounding box, determine which boundary layers have good coverage. Then, based on the zoom pick the right boundary layer for the scale.
